runtime/queries/html_tags/injections.scm

```diff
@@ -39,9 +39,9 @@
       (attribute_name) @_attr
       (#eq? @_attr "type")
       (quoted_attribute_value
-        (attribute_value) @_type)))
+        (attribute_value) @injection.language)))
   (raw_text) @injection.content
-  (#set-lang-from-mimetype! @_type))
+  (#gsub! @injection.language "(.-)/(.+)" "%2"))
```

Comment:
I appreciate the simplicity, but it comes at the cost of lack of `script type="module"` support.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#applying_the_module_to_your_html

Please keep in mind that I didn't do frontend jobs for years (it was an even bigger mess back then), so let me comment the `importmap` case by reference to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#importing_modules_using_import_maps (TLDR: we can probably inject it as `json`).

Reply:
Yes, as I have written, I consider this an acceptable regression _out-of-the-box_. (Not least because it's not actually a mimetype, so the directive was misused.)

You can still make it work by registering -- or, better yet, adding a separate injection query for it.

Reply:
Added the explicit patterns for the no longer covered test cases.

Reply:
It would be no problem for me to add them on the user side, but thanks for adding them here.